src/utils/items.js

```diff
@@ -0,0 +1,20 @@
+import { transformToJSDate, getDaysBetweenDates } from './dates';
+
+export function getItemDaysUntilNextPurchase(item) {
+	const today = new Date();
+	const nextPurchase = transformToJSDate(item.dateNextPurchased);
+
+	return getDaysBetweenDates(nextPurchase, today);
+}
+
+export function sortItems(data) {
+	return data.sort((a, b) => {
```

Comment:
<img width="381" alt="Screenshot 2023-05-11 at 1 01 26 PM" src="https://github.com/the-collab-lab/tcl-56-smart-shopping-list/assets/93563580/6a759323-cf17-468c-ad4f-e351a6be92bd">

Unfortunately on my end, this sort function is not behaving correctly. You can effectively compare a JS Date object using `> <` comparison operators. 
`activeList.sort((a, b) => {
		let currentDate = transformToJSDate(a.dateNextPurchased);
		let nextDate = transformToJSDate(b.dateNextPurchased);

		if (currentDate < nextDate) {
			return -1;
		} else if (currentDate > nextDate) {
			return 1;
		} else {
			// if dates are equal, sort by name
			return a.name.localeCompare(b.name);
		}
	});`
This ensures the date order is first compared and then finally alphabetized.